### PR TITLE
[DFFE-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by Blessing-Hubb**
#Task: Create a database for transportation company
I added sql file containing  queries for transportation database
passengers table
trips table
trips update Table
survival table

### CODE REVIEW

File: submissions/OlusujiBlessing/Databases/transport_database.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+CREATE DATABASE transportation;
+USE transportation;
+CREATE TABLE passengers(
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  full_names VARCHAR(100) NOT NULL,
+  sex ENUM('Male', 'Female') NOT NULL,
```
Comment (adaugochi) [inline submissions/OlusujiBlessing/Databases/transport_database.sql:6]: Don't use `enum` here, kindly use varchar

File: submissions/OlusujiBlessing/Databases/transport_database.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   embarkation_point VARCHAR(100) NOT NULL
+   );
+ALTER TABLE trips ADD COLUMN passenger_class ENUM('1','2', '3');
+
+----This section is for survival table---
+USE transportation;
+CREATE TABLE survival_status(
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  passenger_id INT NOT NULL REFERENCES passengers(id),
+  survival_status ENUM('Yes', 'No') NOT NULL
```
Comment (adaugochi) [inline submissions/OlusujiBlessing/Databases/transport_database.sql:28]: kindly use `tinyint` here

File: submissions/OlusujiBlessing/Databases/transport_database.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  );
+ ----This section is for trips table ---
+USE transportation;
+ CREATE TABLE trips(
+   id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+   passengers_id INT NOT NULL REFERENCES passengers(id),
+   ticket_numbers VARCHAR(50) NOT NULL,
+   trip_fares VARCHAR(10) NOT NULL,
+   cabin_number VARCHAR(5),
+   parentsorchildren_number VARCHAR(20) NOT NULL,
```
Comment (adaugochi) [inline submissions/OlusujiBlessing/Databases/transport_database.sql:17]: this should be an `int`

File: submissions/OlusujiBlessing/Databases/transport_database.sql [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+   spousesorsiblings_number VARCHAR(20) NOT NULL,
```
Comment (adaugochi) [inline submissions/OlusujiBlessing/Databases/transport_database.sql:18]: this should be an `int`

File: submissions/OlusujiBlessing/Databases/transport_database.sql [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+USE transportation;
+CREATE TABLE passengers(
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  full_names VARCHAR(100) NOT NULL,
+  sex ENUM('Male', 'Female') NOT NULL,
+  ages varchar(10)
+  );
+ ----This section is for trips table ---
+USE transportation;
+ CREATE TABLE trips(
```
Comment (adaugochi) [inline submissions/OlusujiBlessing/Databases/transport_database.sql:11]: A column is missing from this table. Kindly check no. 2 of this task given

**Comment by Blessing-Hubb** [comment]
Changes made on lastest transport(0c31056)

**Review by adaugochi — COMMENTED** [review]
You haven't made any changes, kindly check your PR

**Comment by Blessing-Hubb** [comment]
I just pushed another commit, please check

**Review by adaugochi — APPROVED** [review]

**Pull request merged**